sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.5 (2021-03-09)
 Updated `Azure.Communication.Chat` version.
+- Added identifier serialization in Chat
```

Comment:
Identifier serialization is not a public facing concept. We should focus on what effect the developers. For example we can say: Instead of 1 & 2 we can say: Added support for communication identifiers instead of raw strings.
Also all the statements should end with "."